source/server/render-layout.js

```diff
@@ -1,13 +1,12 @@
-export default ({title, rootMarkup, payload}) => {
+export default ({ rootMarkup }) => {
   return `
     <!doctype html>
     <html>
       <head>
-        <title>${ title }</title>
+        <title>React App Hodor</title>
```

Comment:
Shouldn't we pass in the document title? It definitely shouldn't be "React App Hodor".